run_dir/design/clusters_per_lane.html

```diff
@@ -0,0 +1,53 @@
+{% extends "base.html" %}
+
+<!-- Example row of columns -->
```

Comment:
Could we have a chunk saying what this template does please? eg. see the [cronjobs template](https://github.com/SciLifeLab/genomics-status/blob/master/run_dir/design/cronjobs.html#L3)